platform/linuxbsd/x11/display_server_x11.cpp

```diff
@@ -3514,9 +3515,11 @@ void DisplayServerX11::_window_changed(XEvent *event) {
 		//the position in xconfigure is not useful here, obtain it manually
 		int x = 0, y = 0;
 		Window child;
+		XWindowAttributes xwa;
 		XTranslateCoordinates(x11_display, wd.x11_window, DefaultRootWindow(x11_display), 0, 0, &x, &y, &child);
-		new_rect.position.x = x;
-		new_rect.position.y = y;
+		XGetWindowAttributes(x11_display, wd.x11_window, &xwa);
+		new_rect.position.x = x - xwa.x;
+		new_rect.position.y = y - xwa.y;
```

Comment:
`position` is "client rect" origin not a window origin, it should not include decorations and will result in misplaced popups menus/other dropdowns. So this change seem to be wrong, and the issue should be somewhere else.